plugins/hm-stack/inc/api/namespace.php

```diff
@@ -81,6 +81,8 @@ function get_pull_requests() {
  * @return mixed API response, or null for a bad request,
  */
 function query_api( $endpoint, $api_base = HM_STACK_API_URL, $query = [] ) {
+	$endpoint = trim( $endpoint, '/' );
+	$api_base = trailingslashit( untrailingslashit( $api_base ) );
```

Comment:
The `untrailingslashit()` isn't necessary as `trailingslashit()` won't do anything if one if present already.
```suggestion
	$api_base = trailingslashit( $api_base );
```